.circleci/config.yml

```diff
@@ -254,9 +261,103 @@ jobs:
              'until [ `grep "Registered beacons" -s -r /var/log/scion/ | wc -l` -ge 5 ]; do sleep 0.1; done; \
               scmp echo -c 10 -local 20-ffaa:1:4,[127.0.0.1] -remote 20-ffaa:0:1405,[127.0.0.1]'
 
+  upgrade_integration:
+    machine:
+      image: ubuntu-1604:201903-01
+
+    steps:
+      - checkout
+
+      # Start old version of coordinator
+      - run:
+          name: Checkout previous version
+          working_directory: ~/project
+          command: |
```

Comment:
I would prefer to put the checkout into a sibling directory, not a sub.

Reply:
Good point
Done.